src/services/getCharacters.js

```diff
@@ -1,5 +1,12 @@
-export default function getCharacters() {
-  return fetch('https://hp-api.herokuapp.com/api/characters').then((res) =>
-    res.json()
-  )
+import saveToLocal from './saveToLocal'
+
+export default async function getCharacters() {
```

Comment:
The function does not only get characters but also save them in local storage. Maybe reconsinder the name to include both cases